### ISSUE TITLE
slow device flag

### ISSUE THREAD

**Issue by k3tan172** [janoside/btc-rpc-explorer#219]
from what i can tell, version 2.0.1 does not recognise the `BTCEXP_SLOW_DEVICE_MODE=false` line in the .env file anymore. version 2.0.0 does.

to get it working on 2.0.1, you need to have a flag `--slowDeviceMode false` when you start `btc-rpc-explorer`

Is it possible to have the .env file recognise it, rather than requiring the `--slowDevicemode false` flag?

**Comment by janoside** [comment]
Hmm, I just did some testing on 2.0.1 and I'm not seeing this - `BTCEXP_SLOW_DEVICE_MODE=false` is working for me. That's also how the demo site is configured (env var rather than cmd arg). Can you give me any more info about what you're seeing?

**Comment by k3tan172** [comment]
```
ketan@nodeboxtest:~/.config$ btc-rpc-explorer 
Loading env file: /home/ketan/.config/btc-rpc-explorer.env
  btcexp:app Starting BTC RPC Explorer, v2.0.1 +0ms
  btcexp:app Connecting to RPC node at 127.0.0.1:8332 +0ms
  btcexp:app Verifying RPC connection... +7ms
  btcexp:app Loading mining pools config +2ms
  btcexp:app RPC Connected: version=200000 subversion=/Satoshi:0.20.0/, parsedVersion(used for RPC versioning)=0.20.0, protocolversion=70015, chain=main, services=[NETWORK, WITNESS, NETWORK_LIMITED] +18ms
  btcexp:app Loading historical data for chain=main +0ms
  btcexp:app Skipping performance-intensive task: fetch UTXO set summary. This is skipped due to the flag 'slowDeviceMode' which defaults to 'true' to protect slow nodes. Set this flag to 'false' to enjoy UTXO set summary details. +0ms
  btcexp:app Skipping performance-intensive task: fetch last 24 hrs of blockstats to calculate transaction volume. This is skipped due to the flag 'slowDeviceMode' which defaults to 'true' to protect slow nodes. Set this flag to 'false' to enjoy UTXO set summary details. +1ms

```

my .env file contains the `BTCEXP_SLOW_DEVICE_MODE=false` flag.

**Comment by janoside** [comment]
Thanks for the log. I have a handle on this and a fix will be coming soon.

**Comment by ChineseGeek** [comment]
Was this issue resolved? Because i just installed 2.0.1 and got the same result than @k3tan172

**Comment by janoside** [comment]
@ChineseGeek The issue is fixed in the latest code but the code has not yet been included in a release to NPM. I'm working toward that, but trying to get a couple other things included in the next release.
